### PR TITLE
Update_eng_OCRFixReplaceList.xml

### PR DESCRIPTION
**Pull request by diomed**

### CODE REVIEW

File: Dictionaries/eng_OCRFixReplaceList.xml [hunk 1]
```diff
@@ -1195,9 +1195,11 @@
     <Word from="SoBl3lNG" to="SOBBING" />
     <Word from="Sojust" to="So just" />
     <Word from="soldierl" to="soldier!" />
-    <Word from="somethlng" to="somethlng" />
+    <Word from="somethlng" to="something" />
     <Word from="somethlng's" to="something's" />
     <Word from="somez'/7/ng" to="something" />
+    <Word from="somthing" to="something" />
++   <Word from="sumthing" to="something" />
```
Comment (xylographe) [inline Dictionaries/eng_OCRFixReplaceList.xml]: I'm afraid, XmlDocument.Load() (see [Logic\Dictionaries\OcrFixReplaceList.cs](https://github.com/SubtitleEdit/subtitleedit/blob/master/src/Logic/Dictionaries/OcrFixReplaceList.cs#L786)) doesn't like the '+' at the start of line 1202.

Comment (diomed) [inline Dictionaries/eng_OCRFixReplaceList.xml]: this is fucking nightmare. i don't know how is this happening, i didn't write any + signs. 
Can't you modify my commit and then approve it?

Comment (xylographe) [inline Dictionaries/eng_OCRFixReplaceList.xml]: **You** are the admin of *diomed/subtitleedit*.  :smile: 
But, you can copy&paste [this corrected version](https://raw.githubusercontent.com/xylographe/subtitleedit/64b0c1cfeee70c441da1c088e9b76b6c87917266/Dictionaries/eng_OCRFixReplaceList.xml).

Comment (diomed) [inline Dictionaries/eng_OCRFixReplaceList.xml]: @xylographe just merge it from your repository. i don't mind who did it, as long as it's there and it fixes things.
I'll close this now.

**Pull request closed without merging**